assets/javascripts/tests.js

```diff
@@ -90,9 +90,28 @@ function renderTestName(data, type, row) {
         }
         html += dependencyHtml;
     }
-    if (row.comment_count) {
-        html += ' <a href="/tests/' + row.id + '#comments"><i class="test-label label_comment fa fa-comment" title="' +
-            row.comment_count + (row.comment_count != 1 ? ' comments' : ' comment') + ' available"' + '></i></a>';
+    if (row.comment_data) {
+        var bugs = row.comment_data.bugs;
+        for (var i = 0; i < bugs.length; i++) {
+            var bug = bugs[i];
+            var css_class = bug.css_class;
+            var title = bug.title;
+            var url = bug.url;
+            html += '<span id="bug-' + bug.id + '"> ' + '<a href="' + url + '">';
```

Comment:
It would make sense to apply `htmlEscape()` on the variables when generating HTML code.

Reply:
done